src/instructor_clj/core.clj

```diff
@@ -1,34 +1,58 @@
 (ns instructor-clj.core
-  (:require [org.httpkit.client :as http]
-            [stencil.core :as sc]
+  (:require [cheshire.core :as cc]
+            [org.httpkit.client :as http]
             [malli.core :as m]
             [malli.json-schema :as json-schema]
-            [cheshire.core :as cc])
+            [stencil.core :as sc])
   (:import [com.fasterxml.jackson.core JsonParseException]))
 
 
 (defn schema->system-prompt
   "Convert a malli schema into JSON schema and generate a system prompt for responses"
   [schema]
-  (sc/render-string "As a genius expert, your task is to understand the content and provide
-                the parsed objects in json that match the following json_schema:\n
+  (sc/render-string
+   "As a genius expert, your task is to understand the content and provide
+    the parsed objects in json that match the following json_schema:
+    \n\n
+    {{schema}}
+    \n\n
+    Make sure to return an instance of only the JSON, not the schema itself and no text explaining the JSON"
+   {:schema (json-schema/transform schema)}))
 
-                {{schema}}
 
-                Make sure to return an instance of only the JSON, not the schema itself and no text explaining the JSON"
-                    {:schema (json-schema/transform schema)}))
+(defn call-llm-api
```

Comment:
Minor note nothing to be fixed, Let's avoid breaking down functions at this point since we don't know the abstractions yet.

Reply:
Ack